galera/src/replicator_str.cpp

```diff
@@ -820,19 +795,12 @@ ReplicatorSMM::request_state_transfer (void* recv_ctx,
         st_.mark_unsafe();
     }
 
-<<<<<<< HEAD
     // We must set SST state to "wait" before
     // sending request, to avoid racing condition
     // in the sst_received.
-||||||| merged common ancestors
-    send_state_request (req);
-=======
-    GU_DBUG_SYNC_WAIT("before_send_state_request");
-    send_state_request (req);
->>>>>>> release_25.3.30
-
     sst_state_ = SST_WAIT;
-<<<<<<< HEAD
+
+    GU_DBUG_SYNC_WAIT("before_send_state_request");
```

Comment:
Shouldn't this go before the sst_state_ change / comment? Based on the comment those seem to be related, and then this should go before both.

Reply:
fixed